docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,90 @@
+# Installing Kubeapps on BKPR
+
+This guide documents the installation of [Kubeapps](https://kubeapps.com/) on the Kubernetes Production Runtime (BKPR) cluster. You will first install the [Helm](https://www.helm.sh/) package manager to the cluster, followed by the installation of Kubeapps using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps).
+
+The guide assumes you have a Kubernetes cluster with BKPR already installed. The [BKPR installation guide](install.md) documents the process of installing the BKPR client, followed by installing the BKPR server side components on the cluster.
+
+## Step 1: Install the Helm client
+
+Follow the instructions in the [Helm installation guide](https://docs.helm.sh/using_helm/#installing-the-helm-client) to install the latest release of the Helm binary to your machine.
+
+## Step 2: Install Tiller securely
+
+Tiller is the server-side component of Helm that runs inside your Kubernetes cluster. Follow the Helm documentation to [configure role-based access control](https://docs.helm.sh/using_helm/#role-based-access-control) and [install Tiller securely to the cluster](https://docs.helm.sh/using_helm/#using-ssl-between-helm-and-tiller).
+
+## Step 3: Install Kubeapps
+
+Kubeapps is a open-source web-based UI for deploying and managing applications in Kubernetes clusters. It provides you with a dashboard to:
+
+    - Browse Helm charts from public or your own private chart repositories and deploy them to your cluster
+    - Upgrade, manage and delete the applications that are deployed in your cluster
+    - Browse and provision external services from the Service Catalog
+
+Kubeapps can be installed using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps) from the official [Bitnami charts repository](https://github.com/bitnami/charts).
+
+Begin by configuring the Helm client to use the Bitnami charts repository.
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+Install Kubeapps on your Kubernetes cluster with the following command:
+
+```bash
+helm install --tls \
+  --name kubeapps --namespace kubeapps \
+  --set tillerProxy.tls.ca="$(cat ca.cert.pem)" \
+  --set tillerProxy.tls.key="$(cat helm.key.pem)" \
+  --set tillerProxy.tls.cert="$(cat helm.cert.pem)" \
+  --set ingress.enabled=true \
+  --set ingress.certManager=true \
+  --set ingress.hosts[0].name=kubeapps.[YOUR-BKPR-ZONE] \
+  --set ingress.hosts[0].tls=true \
+  --set ingress.hosts[0].tlsSecret=kubeapps-tls \
+  --set mongodb.metrics.enabled=true \
+  bitnami/kubeapps
+```
+
+The command line flags provided in the above command enable the Ingress resource in the Kubeapps chart and also enable TLS support. Once installed, you will be able to generate an access token and then access the Kubeapps dashboard securely (HTTPS) over the Internet at `https://kubeapps.[YOUR-BKPR-ZONE]`.
```

Comment:
The command line flags provided in the above command -> repeats command twice. Maybe "The parameters in the above command"